src/server/scripts/Northrend/Naxxramas/naxxramas.cpp

```diff
@@ -46,19 +89,26 @@ struct npc_frogger_trigger_naxx : public ScriptedAI
         {
             switch (eventId)
             {
-                case EVENT_SUMMON_FROGGER_WAVE:
-                {
-                    std::list<TempSummon*> spawns;
-                    me->SummonCreatureGroup(_nextFroggerWave, &spawns);
-                    if (!spawns.empty())
+                case EVENT_SUMMON_LIVING_POISON:
+                    for (uint8 i = 0; i < MaxLivingPoisons; ++i)
                     {
-                        spawns.front()->GetMotionMaster()->MovePath(10 * NPC_FROGGER + _nextFroggerWave, false);
-                        spawns.front()->UpdateSpeed(MOVE_SWIM); // We need update swim speed because spawn in "water"
+                        if (Creature* slime = DoSummon(NPC_LIVING_POISON, LivingPoisonSpawnPositions[i], 8500/*8s + 500ms*/, TEMPSUMMON_CORPSE_TIMED_DESPAWN))
```

Comment:
yeah, that should be 8s + 500ms for 335 before carbenium comes crawling out of his cave and slaps me again

Reply:
`8500ms` would have been also fine :yum:

Reply:
which do you prefer?
do I change it or leave it like that?

Reply:
Leave it. That's totally fine with me. The non-chrono version actually shouldn't even compile, since those overloads are gone